### DISCUSSION TITLE
The DP potential function is mixed with the EAM potential and the LJ potential

### DISCUSSION THREAD

**Discussion by manbujin** [deepmodeling/dpgen#1526] [Q&A]
Hello，
I trained the TiC potential function with DPGEN, how can I use the DP potential to simulate Ti and C in TiC , and the EAM potential to simulate pure Ti in a model.
Thank you very much for your answer.

**Comment by njzjz** [comment]
It looks like a LAMMPS question. You can use the DP with any other methods available in LAMMPS (with proper LAMMPS commands).
